packages/features/eventtypes/components/tabs/assignment/EventTeamAssignmentTab.tsx

```diff
@@ -422,7 +422,12 @@ export const EventTeamAssignmentTab = ({
     .map((member) => mapUserToValue(member, t("pending")));
   const childrenEventTypeOptions = teamMembers.filter(pendingMembers).map((member) => {
     return mapMemberToChildrenOption(
-      { ...member, eventTypes: member.eventTypes.filter((et) => et !== eventType.slug) },
+      {
+        ...member,
+        eventTypes: member.eventTypes.filter(
+          (et) => et !== eventType.slug || !eventType.children.some((c) => c.owner.id === member.id)
+        ),
+      },
```

Comment:
I don't think this change needed

Reply:
It's required.
For user events where the event slug matches the `EventType` slug:
- If the user is already a member of the event, it ensures that the event is excluded from conflict checks.
- If the user is not a member of the event, the event is included in conflict checks.